webflow-hacks/misc-hacks/12/12.ts

```diff
@@ -0,0 +1,17 @@
+// when DOM ready
+document.addEventListener('DOMContentLoaded', function () {
+  // selector constants
+  const BUTTON_SELECTOR = '[fs-hacks-element="hack-button"]';
+  const TEXT_SELECTOR = '[fs-hacks-element="hack-text"]';
+  // get button and text elements
+  const button = document.querySelector(BUTTON_SELECTOR);
+  const textElement = document.querySelector(TEXT_SELECTOR);
+  // early return if no button
+  if (!button || !textElement) return;
+  // on button click listener
+  button.addEventListener('click', () => {
+    // function - start
```

Comment:
Comments like these:
`// function - start`
`// function - end`

Are unnecessary in my opinion. I think we should just add comments when it's strictly necessary to understand the context. Try to avoid filling the code with redundant comments when possible.
This applies to all hacks, not just this one!

Reply:
okay.